boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -369,28 +378,24 @@ Building
 Flashing
 ========
 #. Run your favorite terminal program to listen for output.
-   Under Linux the terminal should be :code:`/dev/ttyUSB0`. Do not close it.
+   Under Linux the terminal should be :code:`/dev/ttyUSB1`. Do not close it.
 
    For example:
 
    .. code-block:: console
 
-      $ minicom -D /dev/ttyUSB0 -o
-
-   The -o option tells minicom not to send the modem initialization
-   string. Connection should be configured as follows:
+      $ sudo picocom -b 115200 /dev/ttyUSB1
 
-   - Speed: 115200
-   - Data: 8 bits
-   - Parity: None
-   - Stop bits: 1
+   Flash your board using west from the second terminal window. Split first 
+   and second terminal windows to view both of them,because the command in the
+   mec15xxevb_assy6853 board needs to call the FT9FPJZR low instruction first 
+   to execute it, and then call the FT9FPJZR high instruction to run out of 
+   :ref:`hello_world`,therefore we need to use the following command:
 
-#. Flash your board using ``west`` from the second terminal window.
-   Split first and second terminal windows to view both of them.
+    .. code-block:: console
 
-   .. code-block:: console
+      $  ~/flash.sh
```

Comment:
need to specify what is done in this script

Reply:
> need to specify what is done in this script

ok,I will